Podfile

```diff
@@ -10,6 +10,9 @@ platform :osx, '10.12'
 
 # end
 
+# inhibit warning
+install! 'cocoapods', :warn_for_unused_master_specs_repo => false
```

Comment:
`:warn_for_unused_master_specs_repo => false` is not necessary as CDN is default. BUT. You are seeing warnings because previously you also downloaded master repo by using `source 'https://github.com/CocoaPods/Specs.git'`

`pod repo remove master` should do the trick for you, then `pod repo` should show you only `trunk`, which is CDN. :)

Reply:
tried .. didn't work .. will sort it later.